app/src/main/java/ru/prometeydev/movie/data/models/Actor.kt

```diff
@@ -1,7 +1,40 @@
 package ru.prometeydev.movie.data.models
 
+import android.os.Parcel
+import android.os.Parcelable
+import androidx.annotation.DrawableRes
+
 
 data class Actor(
-    val name: String,
+    val id: Long,
+    val name: String?,
+    @DrawableRes
     val avatarDrawable: Int
-)
\ No newline at end of file
+): Parcelable {
```

Comment:
Why parcelable here? 

Check also @Parcelize kotlin plugin

Reply:
ok thanks for the hint, didn't know

Reply:
Great plugin, but I will remove it, since when transferring to a fragment I will use MovieId